env_spec.py

```diff
@@ -47,22 +51,49 @@ def check_input(input_field):
 
 def check_type(input_type):
     """
-    Checks if given type is valid and returns True, else returns False.
-    """
-
+   Checks input type (valid type, valid with default value, restricted choice,
+   restricted choice with default value, comment) for syntax errors.
+   Returs True if syntax is correct, else returns False.
+   """
     if "[" in input_type and "=" in input_type:
-        env_var_type, value = input_type.split("=")
-        value = value.strip()
+        if "#" in input_type:
```

Comment:
Do we need this here, since [you extract the comment below](https://github.com/sourcelair/env-spec-py/pull/15/files#diff-662ae986ce9e28e035956190e3f42976R111) with a regular expression?

Reply:
Yes I need it in all cases, because I want to check if it's syntactically correct every subsring of input string(input_type), except for comments.